coala

```diff
@@ -13,16 +13,13 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import sys
```

Comment:
Although you followed  ```E402 - Fix module level import not at top of file```  but place  the
 ```from coalib.coala import main```
```import sys```  
in the same order which you did below at this point. 
Rest all will be unchanged.

Reply:
@Dhiraj240 So my final sequence would be:
```
from coalib import assert_supported_version

assert_supported_version()

from coalib.coala import main

import sys

if __name__ == '__main__':
    sys.exit(main())
```